doc/nrf/ug_thread_tools.rst

```diff
@@ -115,3 +142,120 @@ The most common wpanctl commands are the following:
+
+1. Open a shell in a PySpinel root directory.
+#. Run PySpinel to connect to the node.
+   The required command depends on if you want to connect to a network co-processor (NCP) node or a radio co-processor (RCP) node.
+
+   Replace the following parameters:
+
+   * *debug_level* - Specifies the debug level, range: ``0-5``.
+   * *ncp_socket_name* - Specifies the location of the socket, for example, :file:`/dev/ACM0`.
+   * *baudrate* - Specifies the baud rate to use.
+     The Thread samples support baud rate ``1000000``.
+
+   Network co-processor (NCP)
+     When connecting to an NCP node, use the following command:
+
+     .. parsed-literal::
+        :class: highlight
+
+        sudo python3 spinel-cli.py -d *debug_level* -p *ncp_socket_name* -b *baudrate*
+
+     For example::
+
+        sudo python3 spinel-cli.py -d 4 -p /dev/ACM0 -b 1000000
+
+   Radio co-processor (RCP)
+     When connecting to an RCP node, you must use the ``ot-ncp`` tool to establish the connection.
+     See :ref:`ug_thread_tools_ot_apps` for more information.
+     To enable logs from the RCP Spinel backend, you must set the ``OPENTHREAD_CONFIG_LOG_OUTPUT`` option to ``OPENTHREAD_CONFIG_LOG_OUTPUT_APP`` when building the ``ot-ncp`` tool.
+     See :ref:`ug_thread_tools_building_ot_apps` for information on how to build the tool with Spinel backend support.
+
+     Use the following command to connect to an RCP node:
+
+     .. parsed-literal::
+        :class: highlight
+
+        sudo python3 spinel-cli.py -d *debug_level* -p './output/posix/bin/ot-ncp spinel+hdlc+uart://\ *ncp_socket_name*\ ?uart-baudrate=\ *baudrate*
+
+     For example::
+
+        sudo python3 spinel-cli.py -d 4 -p './output/posix/bin/ot-ncp spinel+hdlc+uart:///dev/ttyACM0?uart-baudrate=1000000'
+
+.. _ug_thread_tools_ot_apps:
+
+OpenThread POSIX applications
+*****************************
+
+OpenThread POSIX applications allow to communicate with a radio co-processor in a comfortable way.
+
+OpenThread provides the following applications:
+
+* ``ot-ncp`` - Supports :ref:`ug_thread_tools_wpantund` for the RCP architecture.
+* ``ot-cli`` - Works like the :ref:`ot_cli_sample` sample for the RCP architecture.
+* ``ot-daemon`` and ``ot-ctl`` - Provides the same functionality as ``ot-cli``, but keeps the daemon running in the background all the time.
+  See `OpenThread Daemon`_ for more information.
+
+When working with Thread, you can use these tools to interact with the following sample:
+
+* :ref:`ot_coprocessor_sample`
+
+See `OpenThread POSIX app`_ for more information.
+
+.. _ug_thread_tools_building_ot_apps:
+
+Building the OpenThread POSIX applications
+==========================================
+
+Build the OpenThread POSIX applications by performing the following steps:
+
+#. Open a shell in the OpenThread source code directory :file:`ncs/modules/lib/openthread`.
+#. Run one of the following sets of commands, depending on whether you need Spinel backend support.
```

Comment:
Is it correct that the distinction is Spinel backend support or no Spinel backend support?

Reply:
We cannot distinguish it that way. The extra argument: `LOG_OUTPUT=APP` in the command redirects the RCP output (if the Spinel backend was enable) to the "application" provided otPlatLog() in ot-ncp and ot-cli code, possibly even for ot-daemon. Till now I managed only to correctly configure and observe the logs from RCP in PySpinel and wpantund combined with ot-ncp.

Reply:
OK, I tried to fix that section - please check. :)

Reply:
It looks neat now, thanks.